mod/ping.php

```diff
@@ -289,16 +289,30 @@ function ping_init(App $a)
 		}
 
 		if (DBA::isResult($regs)) {
-			foreach ($regs as $reg) {
+			if (count($regs) <= 1 || DI::pConfig()->get(local_user(), 'system', 'detailed_notif')) {
```

Comment:
Why do you check here for the detailled notifications?

Reply:
We talked about grouping the registration requests, I figured we should leave them as they are now for people who enabled the detailed notifications.